ChangeLog.md

```diff
@@ -1,3 +1,31 @@
+## 2017.07.18 - Azure ARM version 0.11.0
+
+* Regened all azure_mgmt_* sdks using AutoRest.1.2.0.0 for release of ARM version 0.11.0
+* Updated [azure_mgmt_analysis_services](https://rubygems.org/gems/azure_mgmt_analysis_services) and replaced configuration object with uri string and added new endpoint to read skus and removed hard coded sku values.(https://github.com/Azure/azure-rest-api-specs/blob/current/specification/analysisservices/resource-manager/readme.md)
```

Comment:
👍 It's very detailed 👍 

Feel free to move the sub-points for each SDK to increase the readability of each change inside one SDK